wdio/specs/homepage.js

```diff
@@ -117,4 +117,25 @@ describe('Municipality homepage', () => {
     const target = browser.getAttribute('.item.action.homepage-teaser:nth-child(4) .content .header a', 'target');
     assert(target == '_blank');
   });
+
+  it('should show QAs only for the chosen user type "residents" and the hebrew language', () => {
+    browser.url('/municipality-1/node/1?user_type=residents&language=he');
+    browser.waitForVisible('div=כמה סייעות יש בגן ילדים?');
```

Comment:
@nader77 
We can update the test after the new design
The next test right after this one is checking business FAQ in not shown on this page L127.